pkg/workflow/invocationcontextimpl.go

```diff
@@ -80,3 +82,7 @@ func (ici *invocationContextImpl) GetEnhancedLogger() *zerolog.Logger {
 func (ici *invocationContextImpl) GetUserInterface() ui.UserInterface {
 	return ici.ui
 }
+
+func (ici *invocationContextImpl) GetRuntimeInfo() RuntimeInfo {
```

Comment:
Obviously this basically proxies `engine.GetRuntimeInfo()`. I was not sure if this is desired or not, would be nice to get clarification.

Reply:
That is totally fine for now. it enables us to add additional logic later if required.